pkg/util/util.go

```diff
@@ -88,3 +93,27 @@ func SendRequest(method, url string, option io.Reader) (*http.Response, error) {
 	}
 	return response, nil
 }
+
+//GetProjectSpec return project spec from a file/flags
+func GetProjectSpec(flags *project.ConfigProject, id string) (*admin.Project, error) {
+	projectSpec := admin.Project{}
+	if len(flags.File) > 0 {
+		yamlFile, err := ioutil.ReadFile(flags.File)
+		if err != nil {
+			return nil, err
+		}
+		err = yaml.Unmarshal(yamlFile, &projectSpec)
+		if err != nil {
+			return nil, err
+		}
+		return &projectSpec, nil
+	}
+
+	projectSpec.Id = id
+	projectSpec.Name = flags.Name
+	projectSpec.Description = flags.Description
+	projectSpec.Labels = &admin.Labels{
+		Values: flags.Labels,
+	}
+	return &projectSpec, nil
+}
```

Comment:
Instead of adding this to the util, this can be member function for project.ConfigProjec

(p *project.ConfigProject) func  GetProjectSpec() {
    // And this can have logic for all the flag reads / flag  read from file
}